src/styles/components/_font-icon.scss

```diff
@@ -0,0 +1,35 @@
+@font-face {
+  font-family: "dso webfont";
```

Comment:
De naam vind ik een beetje knullig. Kunnen we het niet wat fraaier noteren? DSO IconFont?

Reply:
Aangepast

Reply:
Aanpassingen gedaan en ook de iconen in de accordion aangepast.